### DISCUSSION TITLE
慢报：B站新用户开始使用16位uid

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3757] [综合讨论 / General]
https://space.bilibili.com/3461578324183700
https://space.bilibili.com/3461578324183701
https://space.bilibili.com/3461578324183800

**Comment by Tinhone** [comment]
联合投稿 https://www.bilibili.com/video/BV1P44y1K7J7
创作团队：
https://space.bilibili.com/100000000000000
https://space.bilibili.com/100000000000001
https://space.bilibili.com/100000000000002
https://space.bilibili.com/100000000000003

绝版15位uid，含金量极高

**Comment by Geeyun-JY3** [comment]
在开 #4458 时偶然发现的：个人空间搜索结果页好像有个 bug，结果包含的被转发动态所属用户 UID 若为 16 位，在这个页面中这个用户的 URL 中的 UID 是 10 位。
例子：
- https://space.bilibili.com/8047632/search/dynamic?keyword=%E5%9C%B0%E7%90%86
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/13714c5a-5e10-4194-9392-b67bb2c7ad1f)
- 这个例子在 URL 里甚至还有个负号：https://space.bilibili.com/8047632/search/dynamic?keyword=%E9%97%AA%E8%80%80
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/3a000fb5-74dd-4e69-8728-dd081e306a70)

**Reply by the1812** [reply]
盲猜溢出了
